spotbugs-tests/src/test/java/edu/umd/cs/findbugs/sarif/SarifBugReporterTest.java

```diff
@@ -1,10 +1,8 @@
 package edu.umd.cs.findbugs.sarif;
 
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.startsWith;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.hamcrest.Matchers.*;
```

Comment:
@CodingDepot Please no star imports, please resolve those appropriately.  Once done we can merge.  Please also merge in from master to clear changelog conflict.

Reply:
Removed the star imports and resolved changelog conflicts.